build.sbt

```diff
@@ -29,6 +29,8 @@ resolvers += "hmrc-releases-local" at "https://artefacts.tax.service.gov.uk/arti
 
 val bootstrapPlayVersion       = "5.4.0"
 val govTemplateVersion         = "5.68.0-play-26"
+val playFrontendGovUk          = "0.71.0-play-26"
+val playFrontendHmrc           = "0.58.0-play-26"
```

Comment:
Is there any particular reason why you didn't use the latest versions for either of those libraries?